src/vm.rs

```diff
@@ -1411,7 +1428,13 @@ impl<T: GuestStagePageTable> HostVm<T, VmStateInitializing> {
             let mappable = page_tracker
                 .assign_page_for_mapping(page, self.inner.page_owner_id())
                 .unwrap();
-            mapper.map_page_with_measurement(vm_addr, mappable).unwrap();
+            mapper
+                .map_page_with_measurement(
+                    vm_addr,
+                    mappable,
+                    &mut self.inner.attestation_mgr.lock(),
+                )
+                .unwrap();
```

Comment:
Could use a comment about why unwrap() is OK, or else change to unwrap_or_else, etc.

Reply:
The whole `add_measured_pages()` method (together with a few more from the `HostVm` implementation do not carry errors and unwrap. I followed the same pattern here, I think this should be handled through a more general cleanup for those functions, as a separate PR.

Reply:
Yeah, when building the `HostVm` we just unwrap since there isn't anything we can do about the errors.